lib/xmerl/test/xmerl_SUITE.erl

```diff
@@ -45,7 +45,7 @@ all() ->
      xpath_abbreviated_syntax, xpath_functions, xpath_namespaces,
      {group, misc}, {group, eventp_tests},
      {group, ticket_tests}, {group, app_test},
-     {group, appup_test}].
+     {group, appup_test}, {group, format_test}].
```

Comment:
The test group format_test doesn't exist in groups().

Reply:
@lthor : Oops - my apologies - was working in a clean git-tree to submit - missed it.

@Maria-12648430  - thanks for the changes.

I will push another commit on top.

Reply:
Fixed in latest commit.